ToyBox/Classes/MainUI/ActionsRT.cs

```diff
@@ -148,6 +165,90 @@ public static void RerollInteractionSkillChecks() {
             }
         }
 
+        public static void GoToTradeWindow() {
+
+            string[] blockedEtudes = ["725db1ff1322445c8185506f4f6d242e", "6571856eb6c0459cba30e13adc5c6314"];
+            var etudes = EtudesTreeModel.Instance.loadedEtudes;
+
+            foreach (var etude in Game.Instance.Player.EtudesSystem.Etudes.RawFacts) {
+                if (blockedEtudes.Contains(etude.Blueprint.AssetGuid)) {
+                    if (etude.IsPlaying) {
+                        return;
+                    }
+                }
+            }
+
+            var currentArea = Game.Instance.LoadedAreaState.Blueprint;
+            var bridgeArea = ResourcesLibrary.TryGetBlueprint<BlueprintArea>("255859109cec4a042ade1613d80b25a4");
+            
+            if (currentArea == bridgeArea) {
+                var bpa = ResourcesLibrary.TryGetBlueprint<BlueprintAnswer>("d9307ba41f354ad2be00085eca5d0264");
+                (bpa.OnSelect.Actions[0] as Conditional).IfTrue.Actions[0].Run();
+
+            } else {
+                List<Entity> entities = new();
+                List<UnitSpawnerBase.MyData> spawners = new();
+                List<AbstractUnitEntity> units = new();
+                var area = Game.Instance.State.LoadedAreaState;
+                var dialog = ResourcesLibrary.TryGetBlueprint<BlueprintAnswer>("d9307ba41f354ad2be00085eca5d0264");
+                var action = dialog.ElementsArray.OfType<OpenVendorSelectingWindow>().FirstOrDefault();
+
+                try {
+                    var areaState = ResourcesLibrary.TryGetBlueprint<BlueprintArea>("255859109cec4a042ade1613d80b25a4");
```

Comment:
Isn't that already loaded in line 182?

Reply:
Yeah, I can reduce that to one instance - artifact of combining the way that worked on the bridge originally and Kuru's new call which doesn't work on the bridge for some reason I chose not to delve into since I knew how to get it working there.